rust/src/nfs/nfs3_records.rs

```diff
@@ -256,7 +256,7 @@ pub fn parse_nfs3_response_readdirplus_entry(
     let (i, name_contents) = take(name_len as usize)(i)?;
     let (i, _fill_bytes) = cond(name_len % 4 != 0, take(4 - (name_len % 4)))(i)?;
     let (i, _cookie) = take(8_usize)(i)?;
-    let (i, attr_value_follows) = be_u32(i)?;
+    let (i, attr_value_follows) = be_u32(i)?;   // packet with this structure?
     let (i, _attr) = cond(attr_value_follows == 1, take(84_usize))(i)?;
     let (i, handle_value_follows) = be_u32(i)?;
     let (i, handle) = cond(handle_value_follows == 1, parse_nfs3_handle)(i)?;
```

Comment:
I'm not sure I can find a packet for this message type with that structure?
the `READDIR Reply` ends with an entry/ couple of entries and an `EOF` field
no handle field in this message type.

Reply:
Update: looks like `readdirplus Reply` and `readdir Reply` are different messages, and I was testing `parse_nfs3_response_readdir_plus_entry()` against `readdir Reply`.

-- yet should the `readdir reply` message have a separate parser though?

TODO:
- search for a pcap with `readdirplus reply` message to test against.

Reply:
We often do not parse everything that a protocol might provide. Only the important fields and types relevant to our system and users. From what I understand, for anything that is not a particularly parsed type, we simply log the status: https://github.com/OISF/suricata/blob/master/rust/src/nfs/nfs3.rs#L261 and the common fields that come irrespective of the type of procedure and/or version https://github.com/OISF/suricata/blob/master/rust/src/nfs/log.rs#L155

Reply:
`nfs3-01` in SV has this packet type